app/views/products/new_from_git_remote.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render 'header' %>
+
+<%= bootstrap_form_for(@product, as: :product, url: create_from_git_remote_path(type: @type)) do |f| %>
+    <%= render "form_directory", f: f if @product.new_record? %>
```

Comment:
I'm not sure, this was ported from the existing form logic. I'll have to look into it, but at first glance it doesn't appear that it's needed in this view.